### DISCUSSION TITLE
Share installed extensions across different machines

### DISCUSSION THREAD

**Discussion by pheew** [neoclide/coc.nvim#3000] [Q&A] [answered]
I have most of my dotfiles in a git repo and would like to add the coc extensions into the mix. Is there a 'blessed' way to go about this. So basically my question is which files would I need to add?

`~/.config/coc/extensions/package.json` perhaps?

**Comment by fannheyward** [comment, marked as answer]
Checkout `:h g:coc_global_extensions`.

**Reply by pheew** [reply]
Nice, thanks a lot for pointing me in the right direction @fannheyward
